src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -703,7 +681,7 @@ def get_field_domain_offset(
         #
         connectivity_arrays = {
             dace_utils.connectivity_identifier(offset)
-            for offset in dace_utils.filter_connectivity_types(self.offset_provider_type)
+            for offset in dace_utils.filter_connectivity_types(self.offset_provider_type).keys()
```

Comment:
The `keys()` here is technically unnecessary as it is the default when you iterate.

I also prefer doing it, but I was told to avoid it.

Reply:
That explains why it worked in baseline.